streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1086,11 +1096,38 @@ public void recordCacheSensor(Sensor sensor, double count) {
             sensor.record(count);
         }
 
+        @Override
+        public Sensor sensor(String scopeName, String entityName, String operationName, Sensor.RecordLevel recordLevel, String... tags) {
+            return metrics.sensor(sensorName(operationName, entityName), recordLevel);
+        }
+
+        @Override
+        public Sensor sensor(String scopeName, String entityName, String operationName, Sensor.RecordLevel recordLevel, Sensor... parents) {
```

Comment:
One problem here is the `prefix` which so far has been hardcoded in `StreamThread.java`